xonsh/history/sqlite.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+"""Implements the xonsh history backend via sqlite3."""
+import builtins
+import os.path
+import sqlite3
+import threading
+import time
+
+from xonsh.tools import expanduser_abs_path
```

Comment:
importing from outside of package, so import should be `import xonsh.tools as xt`